bin/image-create-openstack.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+IMG_VERSION=${IMG_VERSION:-"v040b1"}
+IMAGE_NAME="kubenow-$IMG_VERSION"
+FILE_NAME="$IMAGE_NAME.qcow2"
+IMAGE_BUCKET_URL="https://s3.eu-central-1.amazonaws.com/kubenow-eu-central-1"
```

Comment:
It needs to default to US for the CI, or it is going to be super slow when importing

Reply:
I  allowed for overriding it with environment variable, and set the environment variable "IMAGE_BUCKET_URL" to US in travis settings web interface